object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -289,20 +305,22 @@ private void doOwnershipTransferToParameters(Set<LocalVarWithTree> newDefs, Node
           if (annotationMirrors.stream()
               .anyMatch(anno -> AnnotationUtils.areSameByClass(anno, Owning.class))) {
             // transfer ownership!
-            newDefs.remove(getAssignmentTreeOfVar(newDefs, local));
+            newDefs.remove(getSetContainingAssignmentTreeOfVar(newDefs, local));
           }
         }
       }
     }
   }
 
-  private void handleReturn(ReturnNode node, ControlFlowGraph cfg, Set<LocalVarWithTree> newDefs) {
+  private Set<ImmutableSet<LocalVarWithTree>> handleReturn(
```

Comment:
Can we just return `void` here now?

Reply:
Yes, we don't need to store the return value anymore.
https://github.com/kelloggm/object-construction-checker/pull/242/commits/73a5d0d6f607756067b47b9b65bd846ff1a9c71d